packaging/installer/kickstart.sh

```diff
@@ -2452,6 +2452,7 @@ parse_args() {
       "--offline-install-source")
         if [ -d "${2}" ]; then
           NETDATA_OFFLINE_INSTALL_SOURCE="${2}"
+          NETDATA_TARBALL_BASEURL="file:///$(cd "$(dirname "${2}")"; pwd)/$(basename "${2}")"
```

Comment:
```suggestion
          NETDATA_TARBALL_BASEURL="file://$(cd "${2}" || exit 1; pwd)"
```

How about just `cd`;`pwd`? Why `dirname`/`basename`? And I guess we need `exit` if fail to `cd`?

Reply:
The `dirname`/`basename` part is mostly habit (it’s required to make the snippet work for arbitrary paths and not just directories).

And adding an `|| exit 1` here gets us essentially nothing. The only way that could fail short of the system having serious hardware or software issues is if the directory is removed after the check that it’s an existing directory two lines up passes.

Reply:
> equired to make the snippet work for arbitrary paths

I don't understand why this is needed here.

> || exit 1 here gets us essentially nothing

Right, but it fixes reviewdog warning.